Quadriga/src/main/java/edu/asu/spring/quadriga/service/impl/workspace/ListWSManager.java

```diff
@@ -324,5 +323,15 @@ public boolean getDeactiveStatus(String workspaceId) throws QuadrigaStorageExcep
             return wsDto.getIsdeactivated();
         return false;
     }
+    @Transactional
+    public boolean getArchiveStatus(String workspaceId) throws QuadrigaStorageException 
+    {
+        WorkspaceDTO wsDto = workspaceDao.getWorkspaceDTO(workspaceId.trim());
+        if (wsDto != null)
+        {

```

Comment:
Please read the code style guide (linked on the course book page) and reformat the code.